devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -300,6 +303,23 @@ def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulat
 	}
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.configurationv2.ConfigurationReport cmd) {
+	def checkVal
+	// the last message sent by configure is a configuration get, so if we get a report, we succeeded in transmission
+	// and if the parameter 1 values match what we expect, then the configuration probably succeeded
+	if (isZipato()) {
+		checkVal = 3
+	} else if (isYale()) {
+		checkVal = state.alarmLength
+	}
+	if (checkVal != null) {
+		state.configured = (checkVal == cmd.scaledConfigurationValue)
+	} else {
+		state.configured = true
+	}
+    return [:]
```

Comment:
I'm guessing the IDE gave you 4 spaces vs a tab? 😛

Reply:
fixed. ty